arrows/pytorch/track.py

```diff
@@ -0,0 +1,171 @@
+# ckwg +28
```

Comment:
This is one very specific tracker. Should we have a name that is more-descriptive than track?